index.html

```diff
@@ -37,6 +36,146 @@ <h4>LET’S CONNECT</h4>
+               no accounts or sign-ups required.</p>
+           <div>
+                <ul class="languages">
+                 <li class="html">html</li>
+                 <li class="css">css</li>
+                 <li class="js">javaScript</li>
+                </ul>
+            </div> 
+         <button type="button">See Project</button>  
+       </div>
+       <div class="card-2">
+        <img src="images/Snapshoot-Porfolio2.png" alt="">
+       
+         <h1>Multi-Post Stories</h1>  
+         <div >
+          <ul class="container">
+              <li><h3>CANOPY</h3></li>
+              <li class="counter1"><img src="images/Counter.png" alt=""> Backend Dev</li>
+              <li class="counter2"><img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+         </div>
+         <p>A daily selection of privately personalized reads; 
+             no accounts or sign-ups required.</p>
+         <div>
+              <ul class="languages">
+               <li class="html">html</li>
+               <li class="css">css</li>
+               <li class="js">javaScript</li>
+              </ul>
+          </div> 
+       <button type="button">See Project</button>  
+     </div>
+     <div class="card-3">
+        <img src="images/Snapshoot-Portfolio-3.png" alt="">
+       
+         <h1>Tonic</h1>  
+         <div >
+          <ul class="container">
+              <li><h3>CANOPY</h3></li>
+              <li class="counter1"><img src="images/Counter.png" alt=""> Backend Dev</li>
+              <li class="counter2"><img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+         </div>
+         <p>A daily selection of privately personalized reads; 
+             no accounts or sign-ups required.</p>
+         <div>
+              <ul class="languages">
+               <li class="html">html</li>
+               <li class="css">css</li>
+               <li class="js">javaScript</li>
+              </ul>
+          </div> 
+       <button type="button">See Project</button>  
+     </div>
+     <div class="card-4">
+        <img src="images/Snapshoot-Portfolio-4.png" alt="">
+       
+         <h1>Multi-Post Stories</h1>  
+         <div >
+          <ul class="container">
+              <li><h3>CANOPY</h3></li>
+              <li class="counter1"><img src="images/Counter.png" alt=""> Backend Dev</li>
+              <li class="counter2"><img src="images/Counter.png" alt=""> 2015</li>
+          </ul>
+         </div>
+         <p>A daily selection of privately personalized reads; 
+             no accounts or sign-ups required.</p>
+         <div>
+              <ul class="languages">
+               <li class="html">html</li>
+               <li class="css">css</li>
+               <li class="js">javaScript</li>
+              </ul>
+          </div> 
+       <button type="button">See Project</button>  
+     </div>
+    </div>
+    </section>
     
+    <section class="about">
+        <div>
+            <h1>About Myself</h1>
```

Comment:
- [ ] kindly make use of headers from `h2` down to `h6` for any other headers apart from your main title of "I’m Zelalem Glad to see you!" in your HTML file.